### ISSUE TITLE
Developer Portal does not by default use your encryptedSummonerId when making calls

### ISSUE THREAD

**Issue by Linnun** [RiotGames/developer-relations#30]
On the developer page you can test-run each endpoint call with your development api key without having to insert it.
Furthermore, if you run an endpoint call that requires you to put in a summoner name, it will also use your own summoner name, if the field is left blank.

The same thing used to work for summonerId and accountId on v3 endpoints, however it currently does not work with encryptedSummonerId and encryptedAccountId for v4 endpoints.

Since the encrypedSummonerId remains the same for you after regenerating a new api key, you should be able to know the correct encryptedSummonerId for the development key, and thus could make the developer portal assume your own encryptedSummonerId if the field is left blank. This is just a little QoL that makes life for developers easier.

**Comment by RiotTuxedo** [comment]
Yep agreed. This is planned, but it wasn't a blocker for releasing v4. Thanks for making an issue, I'll update this thread when it goes live.
